src/index.tsx

```diff
@@ -1,7 +1,7 @@
 import ReactDOM from 'react-dom';
-import { App } from './App';
+import { App, goods } from './App';
 
 ReactDOM.render(
-  <App />,
+  <App goods={goods} />,
```

Comment:
No need to change this file
```suggestion
  <App />,
```

Reply:
I have errors when i do that.